examples/gnrc_minimal/Makefile

```diff
@@ -24,7 +24,7 @@ USEMODULE += gnrc_icmpv6_echo
 USEMODULE += prng_minstd
 
 CFLAGS += -DLOG_LEVEL=LOG_NONE  # disable log output
-CFLAGS += -DGNRC_PKTBUF_SIZE=512 -DGNRC_NETIF_IPV6_ADDRS_NUMOF=2 \
+CFLAGS += -DGNRC_PKTBUF_SIZE=512 -DCONF_GNRC_NETIF_IPV6_ADDRS_NUMOF=2 \
```

Comment:
```suggestion
CFLAGS += -DGNRC_PKTBUF_SIZE=512 -DCONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF=2 \
```
?

Reply:
Whops. Very good catch! Should be fixed here and in one other Makefile.

Reply:
Nice catch